Okta.Xamarin/Okta.Xamarin/OidcClient.cs

```diff
@@ -19,9 +19,9 @@ namespace Okta.Xamarin
 	public partial class OidcClient : IOidcClient
 	{
 		/// <summary>
-		/// The configuration for this Client.  Must be set in the constructor.
+		/// The configuration for this Client.
 		/// </summary>
-		public IOktaConfig Config { get; private set; }
+		public IOktaConfig Config { get; set; }
```

Comment:
I'm curious, why are we changing this? I believe the configuration should remain immutable. Once you create a client with a specific config, how is this gonna work if I change the config afterward?

Reply:
See previous comment (https://github.com/okta/okta-oidc-xamarin/pull/16#discussion_r496137768).  To address your concern I can plan to make this settable only once, however, resetting the config won't necessarily have any adverse affects.